modules/claims_api/app/workers/claims_api/claim_establisher.rb

```diff
@@ -18,6 +19,21 @@ def perform(auto_claim_id) # rubocop:disable Metrics/MethodLength
       form_data = auto_claim.to_internal
       auth_headers = auto_claim.auth_headers
 
+      if auth_headers['va_eauth_birlsfilenumber'].blank? || auth_headers['va_eauth_pid'].blank?
+        # call MPI to get missing values
+        mpi = veteran_from_headers(auth_headers).mpi
+        auth_headers['va_eauth_pid'] = mpi.mvi_response.profile.participant_id
+        auth_headers['va_eauth_birlsfilenumber'] = mpi.mvi_response.profile.birls_id
+
+        # Save IDs and original data
+        auto_claim.auth_headers = auth_headers
+        auto_claim.form_data = orig_form_data
+        auto_claim.save
+
+        # Raise error to trigger a retry
+        raise ClaimsApi::Error::MissingIdException, auto_claim_id
```

Comment:
do you think it's worth raising an error here and just retrying? curious if  you tried to reformat the auto_claim and just continued with this attempt?

Reply:
I mostly did a retry so we'd know/track when that happens.